README.md

```diff
@@ -20,7 +20,72 @@
 
 ### configuration and population
 
+The config file config.py got a lot of interesting setting:
 
+```python
+population_file_base="e2pieces.txt"
+NGEN = 2000
+mutate_inpd=0.01
+selection_ind_value_step=1
+elitism_percentage_start=10
+elitism_percentage_up=4
+gen_modulo_elitism=100
+```
+
+- population_file_base:
+- NGEN:
+
+### Workflows. Running Scripts
+
+  The workflow starts by running the script called main then the script render
+
+  ```bash

```

Comment:
`bash` is not the best rendering option for this code, I think we should use plain `text`